doc/nrf/gs_programming.rst

```diff
@@ -98,8 +99,12 @@ it might return an error indicating a project load failure. For example::
 	The project file <filepath> is invalid.
 	The reported error is 'solution load command failed (1)'
 
-This issue might be caused by a variety of problems, such as incorrectly specified
-project file paths.
+This issue might be caused by a variety of problems, such as incorrectly specified project file paths.
 SES helps you to identify the source of the issue by providing a text output
 with detailed information about the error. Make sure to click **OK** on the
 error pop-up message and then inspect the text output in SES.
+
+.. tip::
+   SES uses the PATH variable to find executables.
+   If you get an error that a tool or command cannot be found, add its location to the PATH variable.
+   For some tools, you can explicitly specify the location under **Tools -> Options** (select the **nRF Connect** tab).
```

Comment:
Do we need to add the WEST path here for macOS?

Reply:
It is currently being discussed on how to improve the user experience on MacOS.
But for time being I believe i would be a good idea to descibe how the user can specify path to `west` using `-DWEST=<path-to-west>`

Reply:
Yeah, that's the safest option as you don't have a PATH variable available in this tab

Reply:
If we don't use the global PATH on macOS, how do we specify where to find mergehex?

Reply:
Added